lsquic/client.nim

```diff
@@ -3,39 +3,72 @@ import ./[errors, connection, tlsconfig, datagram, connectionmanager]
 import ./context/[context, io, client]
 
 type QuicClient* = ref object of RootObj
-  quicContext: ClientContext
   connman: ConnectionManager
-  udp: DatagramTransport
+  tlsConfig: TLSConfig
+  ctx4: ClientContext
+  ctx6: ClientContext
+  udp4: DatagramTransport
+  udp6: DatagramTransport
```

Comment:
Is it necessary to have duplicated ClientContext and DatagramTransport?   newDatagramTransport6 returns a DatagramTransport anyway, and unless i am missing something, once both of them are instantiated, there isnt in practice any difference in usage whether they're ipv6 or ipv4